apps/molgenis-components/src/components/utils.ts

```diff
@@ -224,3 +224,12 @@ export function applyComputed(rows: IRow[], tableMetadata: ITableMetaData) {
     }, {});
   });
 }
+
+// Functions for making sure empty field is null instead of empty for forms.
+export function updateModelValue(object, value) {
+  updateModelValueEmit(object.$emit, value);
+}
+
+export function updateModelValueEmit(emit, value) {
```

Comment:
- These contain ts errors (implicit any type). Do you have your IDE setup correctly? It should show red lines on these functions.
- Is it possible to pass this.$emit into the second function? In that case we could remove the first.
- This comment should not be here, comments double the maintenance of code so should be used only if really necessary.
- I suggest renaming the function to better reflect what they do e.g. `emitModelValueNullSafe`